data/campaigns/Legend_of_Wesmere/utils/low-macros.cfg

```diff
@@ -59,13 +59,16 @@
 #enddef
 
 #define ELVES_SCOUTS
-Elvish Scout, Elvish Rider, Elvish Outrider, Elvish Horse Archer#enddef
+Elvish Scout, Elvish Rider, Elvish Outrider
+#enddef
 
 #define ELVES_STEEL
-Elvish Fighter, Elvish Hero, Elvish Captain, Elvish Marshal, Elvish Champion, Elvish Archer, Elvish Marksman, Elvish Sharpshooter, Elvish Ranger, Elvish Avenger, {ELVES_SCOUTS}#enddef
+Elvish Fighter, Elvish Hero, Elvish Captain, Elvish Marshal, Elvish Champion, Elvish Archer, Elvish Marksman, Elvish Sharpshooter, Elvish Ranger, Elvish Avenger, {ELVES_SCOUTS}
+#enddef
 
 #define ELVES_NON_STEEL
-Elvish Shaman, Elvish Shyde, Elvish Enchantress, Elvish Sorceress, Elvish Sylph, Elvish Druid, Wose, Elder Wose, Ancient Wose, {ELVES_SCOUTS}#enddef
+Elvish Shaman, Elvish Shyde, Elvish Enchantress, Elvish Sorceress, Elvish Sylph, Elvish Druid, Wose, Elder Wose, Ancient Wose, {ELVES_SCOUTS}
+#enddef
```

Comment:
These `#enddef` directives should not be on a new line of their own since these macros are intended to be used inline as an attribute value. (This probably applies to the one up in scenario 5, too.)